module/People/src/People/Controller/MembersController.php

```diff
@@ -132,4 +112,26 @@ protected function getResourceOptions()
 	{
 		return self::$resourceOptions;
 	}
+	
+	public function setEventManager(EventManagerInterface $events)
```

Comment:
Perché non hai messo questi metodi, setEventManager e findOrganization (il cui nome non va bene), in una classe base come concordato?